api/jsonapi.go

```diff
@@ -1016,7 +1028,12 @@ func (i *jsonAPIHandler) POSTInventory(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	for _, in := range invList {
-		err := i.node.Datastore.Inventory().Put(in.Slug, in.Quantity)
+		formatted, err := json.Marshal(in.Variants)
```

Comment:
Why store things as opaque blobs instead of normalized tables? It prevents atomicity, queryability, and data integrity.